components/display/max7219digit.rst

```diff
@@ -0,0 +1,181 @@
+
+Configuration variables:
+------------------------
+
+- **cs_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The pin you have the CS line hooked up to.
+- **num_chips** (*Optional*, integer): The number of chips you wish to use for daisy chaining. Defaults to
+  ``4``.
+- **rotate_chip** (*Optional*, enum): The configuration of how the chips are aligned can be changed. Range
+  is from 0 (the default), 90, 180 up to 270. Each step is rotating the chip display by 90 degrees.
+- **scroll_enable** (*Optional*, boolean): Turn scroll mode on (True).
+- **scroll_speed** (*Optional*, :ref:`config-time`): Set scroll speed in ms. The default is (250 ms)
+- **scroll_delay** (*Optional*, :ref:`config-time`): Set delay of scroll at start of the string in ms.
+  The default is (1000 ms)
+- **scroll_dwell** (*Optional*, :ref:`config-time`): Set the delay of scroll at the end of the string in ms.
+  The default is (1000 ms). This is only used in mode 'STOP'.
+- **scroll_mode** (*Optional*, ): Set the scroll mode. 'CONTINUOUS' = continuously and 'STOP' = stop and reset
+  at end.
+- **intensity** (*Optional*, integer): The intensity with which the MAX7219 should drive the outputs. Range is
+  from 0 (least intense) to 15 (the default).
+- **lambda** (*Optional*, :ref:`lambda <config-lambda>`): The lambda to use for rendering the content on the
+  MAX7219. See :ref:`display-max7219digit_lambda` for more information.
+- **update_interval** (*Optional*, :ref:`config-time`): The interval to re-draw the screen. Defaults to ``1s``.
+- **spi_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`SPI Component <spi>` if you want
+  to use multiple SPI buses.
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+
+.. _display-max7219digit_lambda:
+
+Rendering Lambda
+----------------
+
+The MAX7219 digit is based on the fully fledged :ref:`display-engine`, as it has a concept of individual pixels 8 X 8
+per max7219 chip. In the lambda you're passed a variable called ``it`` as with all other displays. Some "Special"
+commands have been added to the basic display set.
+
+.. code-block:: yaml
+
+    display:
+      - platform: max7219digit
+        cs_pin: D8
+        num_chips: 4
+        lambda: |-
+          it.strftime(0, 0, id(digit_font), "%H:%M", id(hass_time).now());
+          it.image(24, 0, id(my_image));
+          it.line(1, 7, 21, 7);
+    font:
+      - file: "pixelmix.ttf"
+        id: digit_font
+        size: 6
+
+    time:
+      - platform: homeassistant
+        id: hass_time
+
+    image:
+      - file: "smile.png"
+        id: my_image
+
+This is roughly the code used to display the MAX7219 pictured in the image.
+
+Some special MAX7219 digit code can be added as follows:
+
+.. code-block:: yaml
+
+    display:
+      - platform: max7219digit
+        # ...
+        lambda: |-
+          // Print 0 at position 0 (left)
+          it.print(0,0, id(digit_font), "Hello!");
```

Comment:
Do you mean the other print here? The one with the embedded font?

Reply:
I have removed this part as it is already explained in the code block below and does not clarify anything